docs/overrides/home.html

```diff
@@ -0,0 +1,191 @@
+            <div class="mdx-feature__item">
+              <div class="mdx-feature__icon">
+                {% include ".icons/material/account-group.svg" %}
+              </div>
+              <div class="mdx-feature__content">
+                <h2>Community Driven</h2>
+                <p>Developed by the neuroscience community for the community. BIDS continues to evolve based on real-world research needs and feedback.</p>
+              </div>
+            </div>
+          </div>
+
+          <div class="mdx-feature">
+            <div class="mdx-feature__item">
+              <div class="mdx-feature__icon">
+                {% include ".icons/material/tools.svg" %}
+              </div>
+              <div class="mdx-feature__content">
+                <h2>Rich Ecosystem</h2>
+                <p>A growing collection of BIDS Apps, validators, and analysis tools that work seamlessly with BIDS-formatted datasets.</p>
+              </div>
+            </div>
+          </div>
+
+          <div class="mdx-feature">
+            <div class="mdx-feature__item">
+              <div class="mdx-feature__icon">
+                {% include ".icons/material/open-source-initiative.svg" %}
+              </div>
+              <div class="mdx-feature__content">
+                <h2>Open Science</h2>
+                <p>Promotes reproducible research and data sharing through standardized formats that are platform-independent and openly documented.</p>
+              </div>
+            </div>
+          </div>
+
+          <div class="mdx-feature">
+            <div class="mdx-feature__item">
+              <div class="mdx-feature__icon">
+                {% include ".icons/material/check-decagram.svg" %}
+              </div>
+              <div class="mdx-feature__content">
+                <h2>Validation Tools</h2>
+                <p>Built-in validation ensures your datasets comply with BIDS standards, catching issues early and improving data quality.</p>
+              </div>
+            </div>
+          </div>
+
+          <div class="mdx-feature">
+            <div class="mdx-feature__item">
+              <div class="mdx-feature__icon">
+                {% include ".icons/material/brain.svg" %}
+              </div>
+              <div class="mdx-feature__content">
+                <h2>Multi-Modal Support</h2>
+                <p>Supports a wide range of neuroimaging modalities including fMRI, EEG, MEG, iEEG, PET, and behavioral data.</p>
+              </div>
+            </div>
+          </div>
+
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <!-- Stats/Social proof section -->
+  <section class="mdx-container" style="border-top: none;">
+    <div class="md-grid md-typeset">
+      <div class="mdx-stats">
+        <a href="https://openneuro.org/" class="mdx-stats__item" target="_blank" rel="noopener noreferrer">
+          <div class="mdx-stats__number">2000+</div>
+          <div class="mdx-stats__label">Public BIDS Datasets
```

Comment:
```suggestion
          <div class="mdx-stats__number">1,500+</div>
          <div class="mdx-stats__label">OpenNeuro Public Datasets
```

That number is "public" datasets coming specifically from "OpenNeuro".